### ISSUE TITLE
Spatial reverb bus - Wwise

### ISSUE THREAD

**Issue by HDalsfelt** [microsoft/ProjectAcoustics#156]
Hi,

I've added the spatial reverb to our project (UE4.27, WWise 2021.1.7) and ran into some weird problems. When authoring in Wwise it works fine, 12 channels on the spatial reverb and you can hear position when modifying the RTPCs. But when running from in-game it mixes down to two channels on the spatial reverb, and with no audible difference in position. The RTPCs update correctly and the sample project works fine. I wonder if there might be some old Wwise setting in UE that causes this or do you have any other suggestions?

**Comment by drewbitllama** [comment]
Hi HDalsfelt, thank you for reaching out with your issue! It sounds like this could be related to an out-of-date config file... you can try by temporarily renaming the configs in Config/ and re-initing UE; refreshing those may solve your problem. Let me know if that solves it or if we can help you triage it further. :)

**Comment by HDalsfelt** [comment]
I did some tests but I didn't manage to fix the problem. Have any idea what setting might cause the problem? Seems like the number of channels is limited or something. I will continue searching

**Comment by drewbitllama** [comment]
Hi HDalsfelt, how are you bussing the spatial reverb output? Is the output set to Audio Objects? And do you see any error logs that pop up on the UE side (search the log for Acoustics)

**Comment by HDalsfelt** [comment]
Yes, I've set it up exactly as the sample-project. Reverb (Audio objects)-> Headphone mix (audio objects) -> Main mix. The weird thing is that it looks/sound fine when authoring in wwise, but when I trigger same event from ingame the spatial reverb outputs stereo.

No error logs in UE, just a message that it down-mix to mono (but I saw that one in the sample project as well). And the actual reverb-effect sounds nice, just missing the "spatial" part.
